packages/babel-plugin-transform-es2015-block-scoping/test/fixtures/general/hoisting-2/expected.js

```diff
@@ -0,0 +1,87 @@
+var _ref5 = function (j, result, i) {
```

Comment:
Why do we generate these as expressions assigned to a `var` anyway? Is it just easier to generate the AST instead of making a function declaration?